pkg/api/v1alpha1/types.go

```diff
@@ -55,6 +55,9 @@ type DeschedulerPolicy struct {
 
 	// MaxNoOfPodsToTotal restricts maximum of pods to be evicted total.
 	MaxNoOfPodsToEvictTotal *uint `json:"maxNoOfPodsToEvictTotal,omitempty"`
+
+	// MinPodAge prevents recently created pods (within the specified minutes) from being evicted.
+	MinPodAge *uint `json:"minPodAge,omitempty"`
```

Comment:
`v1alpha1` is deprecated, we will be removing this type soon. Better to avoid extending it. Also, a conversion needs to be wired to take the new field into account.

Reply:
Thank you for you comments Jan :)

I did not get this one tbh, I removed `MinPodAge` from `v1alpha1` and move it to `DefaultEvictorArgs` https://github.com/kubernetes-sigs/descheduler/blob/ec7ad04c51929cb50cb6a946df63212c4c80154c/pkg/framework/plugins/defaultevictor/types.go#L25-L39

So if we want to avoid extending `v1alpha1`, I do not know which conversion we need to do.

Reply:
This is perfect :) I meant any new field under v1alpha1 type needs a conversion from v1alpha1 -> v1alpha2 included as well. Nothing else to be done here. Thank you.